libs/README.md

```diff
@@ -0,0 +1,25 @@
+# Libraries
+
+This folder contains the MixedReality-WebRTC libraries for C/C++, C#, and Unity.
```

Comment:
Hmm wouldn't one README per lib be better?

Reply:
Be better, no. Be a good thing in addition, maybe. This is mostly to direct users to the right folder, and is displayed on GitHub when browsing to that folder.

Reply:
It doesn't seem to me that it adds a lot to the main README, but can't hurt I guess!